### PR TITLE
book: Add imports to hot reload chapter

### PR DESCRIPTION
**Pull request by tigerros**

### CODE REVIEW

File: book/src/guides/hot_reload.md [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
 
 ## Setup
 
-Just before launching your app, you need to initialize the hot-reload context:
+Before launching your app, you need to initialize the hot-reload context:
 
 ```rust, no_run
-fn main() {
+use freya::prelude::*;
+use freya::hot_reload::FreyaCtx;
```
Comment (marc2332) [inline book/src/guides/hot_reload.md:11]: I should definitely be exporting `FreyaCtx` in the prelude mmm, I'll change it, we can leave this like this for now

**Comment by marc2332** [comment]
Thanks!! 🔥

**Comment by tigerros** [comment]
A fire emoji for two lines? lol

**Comment by marc2332** [comment]
> A fire emoji for two lines? lol

Small details matter :smile:

**Pull request merged**